software/create_custom_firmware_uf2.md

```diff
@@ -1,12 +1,25 @@
 # Creating a custom firmware.uf2 image for EuroPi
 
+## Compiling the firmware on Windows
+1. If not already installed, install [Docker](https://docs.docker.com/desktop/install/windows-install/)
+1. If not already installed, install [GitHub Desktop](https://desktop.github.com/)
```

Comment:
[quesiton] why is github necessary for this?

Reply:
> [quesiton] why is github necessary for this?

Good point, I guess you could just download the repo as a .zip. I'll remove it from these instructions because the step to clone the repo links to the GitHub instructions anyway